src/retinanet.py

```diff
@@ -345,8 +343,8 @@ def encode(self, boxes, labels, input_size):
             boxes = boxes[max_ids]
 
             loc_xy = (boxes[:, :2] - anchor_boxes[:, :2]) / anchor_boxes[:, 2:]
-            loc_hw = torch.log(boxes[:, 2:] / anchor_boxes[:, 2:])
-            loc_targets = torch.cat([loc_xy, loc_hw], 1)
+            loc_wh = torch.log(boxes[:, 2:] / anchor_boxes[:, 2:])
+            loc_targets = torch.cat([loc_xy, loc_wh], 1)
             cls_targets = labels[max_ids]
 
             cls_targets[max_ious < 0.5] = 0
```

Comment:
@taraspiotr are we planning to ever change this value? If so can we move it somewhere?

Reply:
@i008 hmm, for now we didn't talk about changing this 0.4 and 0.5 thresholds from paper and I'm not sure if we will

Reply:
fair, let's leave it as is